src_c/base.c

```diff
@@ -2028,15 +2038,6 @@ pg_uninstall_parachute(void)
 
 /* bind functions to python */
 
-static PyObject *
-pg_do_segfault(PyObject *self, PyObject *args)
```

Comment:
heh. I didn't even realize this function was here, and can't remember what it was for.

Compiler: "I found a function pg_do_segfault that will segfault." Thanks compiler. Yes, probably we should get rid of it.

Reply:
hehe yeah, compiler was going crazy over a segfaulting function, which for some reason, was *made* to segfault